docs/source/tutorial/02-add-the-graphql-schema.mdx

```diff
@@ -2,21 +2,29 @@
 
-<img src="images/graphiql.png" alt="The GraphQL Playground query explorer" class="screenshot"/>
+<img src="images/sandbox_landing.png" alt="The Sandbox query explorer" class="screenshot"/>
 
-The [schema](https://graphql.org/learn/schema/) defines which GraphQL operations your server can execute. Click **Docs** on the right-hand side to view a list of types you can query (and the types of fields on those types) along with any possible mutations or subscriptions.
+You'll know that this Sandbox instance is pointed at our server because its URL, `https://apollo-fullstack-tutorial.herokuapp.com`, is in the box at the top left of the page. If Sandbox is properly connected, you'll see a green dot: 
 
-<img src="images/graphiql_show_schema.png" alt="GraphQL Playground showing the schema" class="screenshot"/>
+<img src="images/sandbox_green_dot.png" alt="A closeup of the URL box with a dot indicating it's connected" class="screenshot"/>
+
+The schema defines which GraphQL operations your server can execute. At the top left, click the schema icon to get an overview of your schema: 
+
+<img src="images/schema_icon.png" alt="The schema icon to click" class="screenshot"/>
+
+In the **Reference** tab, you can now see a list of all of the things available to you as a consumer of this API, along with available fields on all objects:
+
+<img src="images/sandbox_schema_reference.png" alt="Apollo sandbox showing the schema reference" class="screenshot"/>
 
 ## Download your server's schema
 
-Apollo Android requires a schema to generate typesafe models and code from your queries. There are multiple ways to get a schema. For example, you can download a json schema directly from GraphQL Playground by clicking **Schema > Download** in the right pane.
+Apollo Android requires a schema to generate type-safe models and code from your queries. There are multiple ways to get a schema. For example, you can go to the SDL tab and download the raw SDL schema using Apollo Studio Sandbox. 
 
-In this tutorial, we will use the `apolloDownloadSchema` task that is created by the plugin automatically. Since GraphQL supports [introspection](https://graphql.org/learn/introspection/), this will work with any GraphQL endpoint that has introspection enabled.
+In this tutorial, we will use the `apolloDownloadSchema` Gradle task that is created by our plugin automatically. Since GraphQL supports [introspection](https://graphql.org/learn/introspection/), this will work with any GraphQL endpoint that has introspection enabled.
 
-From the root of the project, run the following:
+From the root of the project, run the following in Android Studio's Terminal tab:
 
 ```shell:title=(shell)
 mkdir -p app/src/main/graphql/com/example/rocketreserver/
```

Comment:
We can remove the mkdir, it's [done automatically by the plugin now](https://github.com/apollographql/apollo-android/blob/da2bb95194a73c3168b76a8e06a1429553e30ffd/apollo-gradle-plugin/src/main/kotlin/com/apollographql/apollo3/gradle/internal/ApolloDownloadSchemaTask.kt#L117)

```suggestion
```shell:title=(shell)
```